SQF/dayz_code/stringtable.xml

```diff
@@ -13092,7 +13092,7 @@
 			<Czech>%1 je omezeno.</Czech>
 		</Key>
 		<Key ID="STR_EPOCH_PLAYER_316">
-			<English>You may only access friends' gear in this area.</English>
+			<English>You may only access friends or group members gear in this area.</English>
 			<German>Du kannst hier nur auf das Inventar von Freunden zugreifen.</German>
```

Comment:
Can you  update the german string with that?

<German>Du kannst hier nur auf das Inventar von Freunden oder Gruppenmitgliedern zugreifen.</German>